docs/runbooks/DeprecatedRHEL6Vm.md

```diff
@@ -0,0 +1,26 @@
+# DeprecatedRHEL6Vm
+
+## Meaning
+
+This alert fires when a running virtual machine is based on RHEL6 template.
```

Comment:
@fossedihelm I think you are missing that this template will not be supported in the next release